lib/phlex/context.rb

```diff
@@ -44,7 +44,7 @@ def _standard_element(content = nil, _name: nil, **kwargs, &block)
       name = _name ||= __callee__.name
 
       @_target << Tag::LEFT << name
-      _attributes(kwargs)
+      _attributes(kwargs) if kwargs
```

Comment:
can `kwargs` ever be nil?

Reply:
Oh good point.

Reply:
still might be worth doing `if kwargs.any?` but I wouldn't be surprised if that was slower than falling through to the cache for an empty hash.

Reply:
This turned into a huge performance gain. See #81